Dockerfile

```
@@ -0,0 +1,14 @@
+FROM node:16.17.0-alpine3.15 AS generate
+LABEL maintainer="ezTeam <ezteam@couperin.org>"
+
+EXPOSE 80
+
+WORKDIR /usr/src/app
+
+COPY package*.json /usr/src/app/
+RUN npm ci 
+COPY . /usr/src/app
+RUN npm run build
```

@nojhamster any thoughts on this ? ([`node_modules` is in `.dockeringore`](https://github.com/ezpaarse-project/ezpaarse-web-home/blob/ee3a5c2c02274f278c4f658330c0007be2e293e3/.dockerignore))

```suggestion
COPY . .
RUN npm ci 
RUN npm run build
```